src/utils/SessionHelper.ts

```diff
@@ -0,0 +1,39 @@
+import config from "../config";
```

Comment:
Does this class need a test file?

Reply:
It's mainly for stubbing the session locally, I wouldn't have thought it necessary.

Reply:
Ok, if the code will be removed before release then that's fine